doc/contributing/releases.md

```diff
@@ -428,6 +428,12 @@ and substitute this node version with
 sed -i "s/REPLACEME/$VERSION/g" doc/api/*.md
 ```
 
+For macOS use
+
+```console
+sed -i "" "s/REPLACEME/$VERSION/g" doc/api/*.md
+```
```

Comment:
I've noticed that we've been using the `console` syntax highlighter for the commands here, but GitHub highlights the entire command as blue no matter what unless the commands start with a `$`, `#`, `>`, etc etc. Is there a reason we aren't using `shell` to properly highlight these commands?

Reply:
+1, it should either be 

````markdown
```console
$ sed -i "" "s/REPLACEME/$VERSION/g" doc/api/*.md
```
````

or 

````markdown
```bash
sed -i "" "s/REPLACEME/$VERSION/g" doc/api/*.md
```
````

The linter enforces we use `bash` instead of `shell`, but those are aliases in `highlight.js` so that should not matter.

Reply:
Thanks! Fixed in 0f1ef6125a29305be24461e210aaff41fb77eb41